CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Revision history for cached-io
 
+## 1.3.0.0
+
+- Caching functions previously returned `m (t a)`, but it was easy to accidentally use `join` when `m` and `t` were the same monad (eg. `IO (IO a)`), and not get any caching at all. These functions now use a `Cached` newtype for `t a` to make it harder to do this.
```

Comment:
Can you please make it more obvious that this is a breaking change and provide migration instructions?